CHANGELOG.md

```diff
@@ -8,7 +8,8 @@
 #### Fixes
 
 * [#416](https://github.com/ruby-grape/grape-swagger/pull/416): Support recursive models - [@lest](https://github.com/lest).
-* [#418](https://github.com/ruby-grape/grape-swagger/pull/418): Replaced github ref to rubygems for external gems - [@Bugagazavr](https://github.com/Bugagazavr).
+* [#419](https://github.com/ruby-grape/grape-swagger/pull/419): Replaced github ref to rubygems for external gems - [@Bugagazavr](https://github.com/Bugagazavr).
+* [#420](https://github.com/ruby-grape/grape-swagger/pull/420): Avoid empty hashes, becuase this is a cause of some errors in swagger-ui - [@Bugagazavr](https://github.com/Bugagazavr).
```

Comment:
I think this should be more specific, spell out the error that it avoids? 

Also, if this wasn't a problem in a previous release I would just remove this line.